__tests__/itinerary.test.js

```diff
@@ -0,0 +1,20 @@
+
+const Itinerary = require('../src/itinerary.js');
+const Port = require('../src/port.js');
+const Ship = require('../src/ship.js');
```

Comment:
The Ship is not used in this file, therefore it shouldn't be imported, you can delete it.